Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -698,80 +797,163 @@ function Add-MetadataAndStructures() {
         [Parameter(Mandatory = $true)]
         [PSobject] $Config
     )
-
     $roleAdded = Add-EdwAdminRole -RoleName "DataProcessingServiceUser" -Config $Config
 
-    # POST Terminology data marts to MDS
-    Write-DosMessage -Level "Information" -Message "Creating Terminology metadata. This will take several minutes."
-    $terminologyDataMartId = Invoke-PostToMds -Config $Config -name "Terminology" -path ".\Terminology.json"
-
-    Write-DosMessage -Level "Information" -Message "Creating SharedTerminology metadata. This will take several minutes."
-    $sharedTerminologyDataMartId = Invoke-PostToMds -Config $Config -name "SharedTerminology" -path ".\SharedTerminologyFiveNPEs.json"
-
-    # POST executions 
-    Write-DosMessage -Level "Information" -Message "Creating physical tables for Terminology and SharedTerminology data marts"
-    $terminologyBatchExecutionId = Invoke-PostToDps -Config $Config -dataMartId $terminologyDataMartId
-    $sharedTerminologyBatchExecutionId = Invoke-PostToDps -Config $Config -dataMartId $sharedTerminologyDataMartId
+    $terminologyExists = Test-DataMartExists -Name "Terminology" -Config $Config
+    $sharedTerminologyExists = Test-DataMartExists -Name "SharedTerminology" -Config $Config
 
-    # Poll batch executions for 5 minutes to determine if they've been successful
-    $wasSuccessful = Invoke-PollBatchExecutions -Config $Config -terminologyBatchExecutionId $terminologyBatchExecutionId -sharedTerminologyBatchExecutionId $sharedTerminologyBatchExecutionId
+    if (($terminologyExists -eq $false) -and ($sharedTerminologyExists -eq $false)) {
+        if(!$terminologyDataMartId) {
```

Comment:
Lines 806-814 should be removed

Reply:
yup, bad merge. I think I have resolved it and am fixing it